weathereye/cli.py

```diff
@@ -1,16 +1,44 @@
 """Console script for weathereye."""
 import sys
+from platform import system
+from distro import id, version
 import click
 
 
-@click.command()
+@click.group()
 def main(args=None):
-    """Console script for weathereye."""
-    click.echo("Replace this message by putting your code into "
-               "weathereye.cli.main")
-    click.echo("See click documentation at https://click.palletsprojects.com/")
+    """weathereye command-line interface"""
     return 0
 
 
+# WeatherEye install command group
+@main.group()
+def install():
+    """WeatherEye install command"""
+    pass
+
+
+# command to install surface
+@install.command()
+def surface():
+    """Command to install SURFACE CDMS"""
+
+    # check if OS is supported before install
+    host_system = system().lower()
+    host_name = id()
+    host_version = version()
+
+    if host_system != 'linux':
+        
+        click.echo(f"{("macOS" if host_name == "darwin" else host_name), host_version} is not supported for surface CDMS")
```

Comment:
I get an error here:
```
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/codespace/.local/lib/python3.10/site-packages/weathereye-0.1.0-py3.10.egg/weathereye/cli.py", line 33
    click.echo(f"{("macOS" if host_name == "darwin" else host_name), host_version} is not supported for surface CDMS")
                    ^^^^^
SyntaxError: f-string: unmatched '('```
```

Try changing the outer quotes from double quotes (") to single (')?

Reply:
Perhaps you could also add "currently", e.g.
"is not **currently** supported for SURFACE CDMS"

Reply:
The problem should be solved now Ian.